docs/openapi3.yaml

```diff
@@ -2277,3 +2417,33 @@ components:
         percentage:
             type: number
             description: benchmark percentage weight
+    webhook:
+      type: object
+      required:
+        - name
+        - webhook_url
+        - events
+      properties:
+        name:
+          type: string
+          description: Webhook name
+        webhook_url:
+          type: string
+          description: Webhook url to post events
+        events:
+          description: list of events which will trigger the webhook
+          type: array
+          items:
+            $ref: '#/components/schemas/webhooks_types'
+
+    webhooks_types:
+      type: string
+      enum:
+        - started
+        - first_results
```

Comment:
i think instead of `first_results` we should add a resource name `api_failure`. `first_results` are mostly used to see that the test is written correctly and we don't get 5xx and other errors related to the test itself.

Reply:
fixed

Reply:
fixed